src/lib/tabs/tab-header.ts

```diff
@@ -230,9 +271,25 @@ export class MatTabHeader extends _MatTabHeaderMixinBase
     });
   }
 
+  ngAfterViewInit() {
+    // We need to handle these events manually, because we want to bind passive event listeners.
```

Comment:
I'd expand the comment like
```ts
// Register event handlers for scrolling tabs while holding down the pagination buttons.
// We register these events manually because we want to bind *passive* event listeners.
```